src/multi_vector_simulator/utils/constants.py

```diff
@@ -38,6 +38,11 @@
 REPORT_FOLDER = "report"
 ASSET_FOLDER = "assets"
 
+# variables used for the pdf report parser
+ARG_PDF = "print_report"
+ARG_REPORT_PATH = "report_path"
+PATH_SIM_OUTPUT = "output_folder"
```

Comment:
What is `PATH_SIM_OUTPUT`? I thought this is the whole simulation output. Why is it then defined below the definition for `variables used for the pdf report parser`?

Reply:
This is the name of the dict key which stores the user input for `path to the simulation result json file 'json_with_results.json`, I agreed I should add `ARG_` in front of it for consistency

Reply:
Wait... is it input now? Then why is it called `OUTPUT`? And is this specific to the report or for the whole MVS simulation?

Reply:
Fixed in 39def3874e999abc8b1ccd1ec6477484bbffc85a